### ISSUE TITLE
Support arbitrary symbol in matching pair command

### ISSUE THREAD

**Issue by AndreasArvidsson** [pokey/cursorless-vscode#335]
`clear inside symbol dot`

**Comment by pokey** [comment]
Yeah makes sense to me, though I'm not sure I'd use "symbol" because that would be awkward when you don't prefix with "inside".  I don't have a great alternative though.  Maybe one of the following instead:

- "between".  The problem with this one though is that the target is inclusive, which is not implied by "between", and I believe we already use "between" as range connective
- "extent"
- "range"
- "united"
- "paired"
- "guarding"
- "flank"
- "flanking"
- "surrounding"

**Comment by pokey** [comment]
Would also be good to support the same grammar for wrapping.  So if you can say "take surrounding pipe", you should be able to say "surrounding pipe wrap air"
